e2e/README.md

```diff
@@ -16,6 +16,28 @@ sure the new VM's kubelet is posting NodeReady to the cluster's apiserver, and t
 run on it. Lastly, a set of validation commands are remotely executed on the VM to ensure its live state (file
 existsnce, sysctl settings, etc.) is as expected.
 
+```mermaid
+sequenceDiagram
```

Comment:
This is cool. Didn't know Markdown has this feature

Reply:
It's more a GitHub feature.

I think ADO wiki supports it as well (but not ADO code viewer)